### ISSUE TITLE
LAI simulation is large for grassland and planthydro cannot be used

### ISSUE THREAD

**Issue by niuhanlin** [NGEET/fates#919]
When I run FATES-SP mode, LAI appears cone shaped. And when I run fixed biogeography mode , the LAI is trapezoidal(It grows exponentially and then linearly).
My purpose is to run FATES( fixed biogeography mode), but the current model has some problems with LAI:

        1.The growth period and the decline period are too fast.

        2.LAI remained unchanged during the period of vigorous growth from July to September. 

       3.The LAI maximum is too large for the grass. According to the literature and MODIS satellite data, 3.5 in SP-mode is relatively normal.

![TLAI_in_maqu_fates_sp_spinup_200years clm2 h0 2012](https://user-images.githubusercontent.com/66841974/196129697-06a498b8-63b4-4068-a17b-d185ffd44f04.png)
![TLAI_in_maqu_fates_sp_spinup_200years_again clm2 h](https://user-images.githubusercontent.com/66841974/196129777-8fd0a209-b11c-4818-816f-6fa941afe5ab.png)

So I think it's important to modify the parameters of the allometric assignment curve for grasslands.
Does anyone have any suggestions for that?   Is there a realistic parameter for the grass?

Another issue is that currently the program will report an error when use_fates_planthydro =.true. is set.
[cesm.log.txt](https://github.com/NGEET/fates/files/9798526/cesm.log.txt)
[lnd.log.txt](https://github.com/NGEET/fates/files/9798528/lnd.log.txt)

using version: [sci.1.59.7_api.24.1.0]

**Comment by rosiealice** [comment]
I agree part of the bias with lai may come from parametric issues. ,y suspicion here is that the lai is reaching the 'cap' imposed by the lai trimming mechanism, and losing any leaf layers in negative carbon balance. The reduce that cap one should either decrease productivity or increase turnover. One simple thing would be to modify root and /or lead turnover rates and see what impact this has. 

To reduce the rate of 'ramp up' is more complicated but probably results from npp being too high. You could look at the impact of vcmax to check whether that reduces the leaf biomass increase. 

Senescence is more complex and probably is more related to the phenology model not capturing the processes of leaf aging and late summer senescence across the riversity of phenological strategies we aren't capturing... 

The SP output should be the same as the surface dataset inputs, with all the caveats of spatial averaging and satellite data products.

Hope that helps! 
Rosie
